porch/controllers/packagevariantsets/pkg/controllers/packagevariantset/packagevariantset_controller.go

```diff
@@ -76,13 +81,21 @@ func (r *PackageVariantSetReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, nil
 	}
 
+	var conditions []metav1.Condition
```

Comment:
When we work with conditions, there are a few things we should do to make it easier for clients to use them:
- Always set all the conditions on a resource for every reconcile. So for the `Valid` condition, we should make sure we set it to `True` whenever there are no validation issues.
- We should make sure the `LastTransitionTime` only changes when the value of the `Status` field changes. So we shouldn't recreate the conditions slice from scratch on every reconcile, but update the existing conditions.

Reply:
Thanks for the explanation! I'm now using the apimachinery library to update the condition which should take care of `LastTransitionTime`, and made sure that the `Valid` condition is set to true when everything looks good.